prime-router/src/main/kotlin/history/db/ReportGraph.kt

```diff
@@ -164,7 +164,16 @@ class ReportGraph(
                     ITEM_LINEAGE.CHILD_REPORT_ID.`as`(STARTING_REPORT_ID_FIELD)
                 )
                     .from(ITEM_LINEAGE)
-                    .where(ITEM_LINEAGE.CHILD_REPORT_ID.`in`(reportIds))
```

Comment:
This appears to be the culprit for why it was running so much slower in the API as I was typically testing the queries this way naively assuming that the impact would be the same...